### PR TITLE
Trim whitespace in `./Misc/contentHash/dotnetRuntime/*`

### PR DESCRIPTION
**Pull request by TingluoHuang**

### CODE REVIEW

File: src/Test/L0/PackagesTrimL0.cs [hunk 1]
```diff
@@ -273,5 +273,29 @@ public async Task RunnerLayoutParts_CheckExternalsHash()
                 Assert.True(string.Equals(hashResult, File.ReadAllText(externalsHashFile).Trim()), $"Hash mismatch for externals. You might need to update `Misc/contentHash/externals/{BuildConstants.RunnerPackage.PackageName}` or check if `hashFiles.ts` ever changed recently.");
             }
         }
+
+        [Fact]
+        [Trait("Level", "L0")]
+        [Trait("Category", "Common")]
+        public Task RunnerLayoutParts_ContentHashFilesNoNewline()
```
Comment (TingluoHuang) [inline src/Test/L0/PackagesTrimL0.cs:280]: add a test to make sure we don't add any whitespace in the future...

**Review by luketomlinson — APPROVED** [review]
